src/reducers/index.js

```diff
@@ -55,15 +55,25 @@ const authorization = (
       return {
         ...state,
         ...{
-          isAuthorized: false,
+          isAuthorized: 0,
           authorizationStarted: false,
         },
       };
+    case AUTHORIZATION.PENDING:
+      return {
+        ...state,
+        ...{
+          isAuthorized: 1,
+          authorizationStarted: true,
+          userInfo: action.payload.userInfo,
+          ssoGroup: action.payload.ssoGroup,
+        },
+      };
     case AUTHORIZATION.SUCCESS:
       return {
         ...state,
         ...{
-          isAuthorized: true,
+          isAuthorized: 2,
```

Comment:
i think calling the key `isAuthorized` would infer that it is a boolean value, since it holds more of a 'state' why don't we rename is something like `authCode` or something similar? What do you think?

Reply:
sure